src/os/layer/vector.js

```diff
@@ -428,6 +428,10 @@ os.layer.Vector.prototype.getFASet = function() {
     icons.push(os.ui.Icons.FILTER);
   }
 
+  if (plugin.im.action.feature.Manager.getInstance().hasActiveActions(/** @type {os.source.Request} */ (source))) {
```

Comment:
The `plugin` namespace should never be referenced outside of other `plugin` files. In this instance, this should instead call `os.im.action.ImportActionManager.getInstance()` as the feature action plugin version extends and replaces that singleton.

I would also recommend moving the `hasActiveActions` function to the `ImportActionManager` as it's a handy utility to have on the core singleton.